tests/test_aiocassandra.py

```diff
@@ -95,12 +107,52 @@ def new_patch_factory():
 
     with caplog.at_level(logging.ERROR):
         await asyncio.sleep(0.1, loop=loop)
-        assert len(caplog.records) == 0
+
+    assert len(caplog.records) == 0
 
     with pytest.raises(asyncio.CancelledError):
         await fut
 
 
+@pytest.mark.asyncio
+async def test_execute_futures_simple(cassandra):
+    cql = 'SELECT now() as now FROM system.local;'
+
+    ret = []
+
+    async for row in cassandra.execute_futures(cql):
+        ret.append(row)
+
+    assert len(ret) == 1
+
+    assert isinstance(ret[0].now, uuid.UUID)
+
+
+@pytest.mark.asyncio
+async def test_execute_futures_simple_statement_empty(cassandra):
+    cql = 'SELECT * FROM system_schema.types;'
+    statement = SimpleStatement(cql, fetch_size=1)
+
+    # this tests show that there nothing found and no one records is executed
```

Comment:
It would be more readable if you ever drop 3.5, like this:
```python
assert len(_ async for _ in cassandra.execute_futures(statement)) == 0
```

Reply:
I'd like to keep 3.5